Troubleshooting_Ipv6.md

```diff
@@ -0,0 +1,273 @@
+# IPv6 Disconnected Install Troubleshooting
+
+In this document we will analyze common scenarios of errors/failures which can cuase the cluster installation to fail. 
+
+## Common failures Basic checks before Installing
+
+Below are some precautions we should follow before even attempting the OCP4.3 Ipv6 deployments. 
```

Comment:
OCP 4 IPv6 deployments

Don't pin this doc to a minor release unless we want to continuously update this doc

Reply:
Ok, sure adding the changes